sequencer.c

```diff
@@ -380,7 +380,10 @@ static void print_advice(struct repository *r, int show_hint,
 		else
 			advise(_("after resolving the conflicts, mark the corrected paths\n"
 				 "with 'git add <paths>' or 'git rm <paths>'\n"
-				 "and commit the result with 'git commit'"));
+				 "and commit the result with 'git commit'\n"
+				 "You can also use:\n"
+			 	 "    git [cherry-pick | revert] --skip\n"
```

Comment:
Looking at the code, I guess `rebase -i` can also take the control in these areas. More research is needed though 😅.

Reply:
For what it's worth, I think it would also be fine to omit this extra mention of "you can use `--skip`" here, and instead just leave the message as it is.

Reply:
Yes, I think leaving it out here is fine.  One place that does need an updated advice message which wasn't included in your patch, is in builtin/commit.c, the empty_cherry_pick_advice_* messages.

Reply:
@newren I think I have changed them [here](https://github.com/r1walz/git/blob/8b0d662b655afbd074e8b7572863a3b0e2ce9207/builtin/commit.c#L55-L71), are you talking about something else?

Reply:
Oh, indeed, they're part of your third commit and they look good.  Missed them somehow previously; sorry about that.

Reply:
I guess I'll just squash my commis and send them over here. Right now, I'm reading tests in `t3510` to see where shall I insert my tests.